packages/conventional-commits-parser/rollup.config.js

```diff
@@ -0,0 +1,58 @@
+import { swc } from 'rollup-plugin-swc3'
+import { nodeResolve } from '@rollup/plugin-node-resolve'
+import shebang from 'rollup-plugin-add-shebang'
+import nodeEsm from '@trigen/browserslist-config/node-esm'
+import pkg from './package.json' assert { type: 'json' }
+
+const extensions = ['.js', '.ts']
+const external = _ => /node_modules/.test(_) && !/@swc\/helpers/.test(_)
+const plugins = targets => [
+  nodeResolve({
+    extensions
+  }),
+  swc({
+    tsconfig: false,
+    jsc: {
+      parser: {
+        syntax: 'typescript',
+        tsx: true
+      },
+      externalHelpers: true
+    },
+    env: {
+      targets
+    },
+    module: {
+      type: 'es6'
+    },
+    sourceMaps: true
+  })
+]
+
+export default [
```

Comment:
q: do we really need rollup to build this project?

Reply:
good point, tsc should be enough